### ISSUE TITLE
A11y bug: Filter headings missing focus state

### ISSUE THREAD

**Issue by andycochran** [HHS/simpler-grants-gov#8416]
### Summary

**[Loom](https://www.loom.com/share/2c4386d17e1041e88d27351370129167)**

When tab navigating to filter accordions, some of them do not get the appropriate `:focus` styles. It seems this is a result of differing classes, e.g.… 

This `.usa-accordion__heading` gets a focus style: 
```
<h2 class="usa-accordion__heading overflow-visible"><button type="button" class="usa-accordion__button" aria-expanded="true" aria-controls="opportunity-filter-fundingInstrument" data-testid="accordionButton_opportunity-filter-fundingInstrument">Funding instrument</button></h2>
```

This one does not: 

```
<h2 class="usa-accordion__heading maxh-mobile-lg overflow-auto position-relative"><button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="opportunity-filter-eligibility" data-testid="accordionButton_opportunity-filter-eligibility">Eligibility</button></h2>
```

(When I recorded the Loom, I thought this might be a Safari issue (as it handles `:focus-visible` differently), but seems the bug is there in Chrome too.) 

### Reproducibility

I tried multiple times and saw the bug every time

### How to reproduce the bug

- Open the filter pane
- Use Tab to navigate with the keyboard to the various filters
- Note how filters with specific classes no not get the appropriate `:focus` states

### Browser

Safari

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by ErinPattisonNava** [comment]
Just confirming that you turned on highlighting specifically in Safari.  Alas, it isn't on by default: https://www.a11y-collective.com/blog/how-to-activate-keyboard-navigation-on-macos/

see section "Highlight each focusable item in Safari"

**Comment by andycochran** [comment]
Yeah. I was sus that this was a Safari issue. They've made… [choices](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/button#clicking_and_focus). But checked it's present in Chrome too

**Comment by ErinPattisonNava** [comment]
Thanks for confirming @andycochran !

@chelsea-nava and @eric-serrato95 note that this does put us out of ADA compliance on our largest use case, so I'd recommend prioritizing work on this.

**Comment by andycochran** [comment]
Should be a pretty easy fix of standardizing the heading classes. IDK why some of them are different. And I don't think they need to be.

**Comment by chelsea-nava** [comment]
Thanks for the callout - yes let's prioritize getting back into compliance and making those updates. We have a good amount of tweaks being made so lowest lift solve would be best

**Comment by eric-serrato95** [comment]
@andycochran would you be able to implement these changes? or should we try to carve some FE capacity on our side to address this? 
CC: @ErinPattisonNava @chelsea-nava

**Comment by andycochran** [comment]
I could make a change, but I'd at least want to sync w/ the Find team to see if there's any reason for the extra utility classes. Can we discuss this at an upcoming refinement session?

**Comment by andycochran** [comment]
I think this might be a result of the Truss component applying `className` to both the `headingClasses` and the `contentClasses`. 

https://github.com/trussworks/react-uswds/blob/15d032a3417d75b55f6cf08cb13d1a7435c4e7f1/src/components/Accordion/Accordion.tsx#L32-L37

So when we've applied utility classes intended to make the content scrollable, they also try to make the heading scrollable.

**Comment by andycochran** [comment]
Assinging @eric-serrato95 for UAT

**Comment by andycochran** [comment]
Change is live at https://staging.simpler.grants.gov/search

**Comment by eric-serrato95** [comment]
UAT ✅  closing ticket @andycochran
